deploy/systemd/dmq@.service

```diff
@@ -0,0 +1,28 @@
+[Unit]
+Description=Distributed Message Queue instance %I
+AssertPathExists=/srv/dmqnode/dmqnode-%i
+Requires=network-online.target local-fs.target
+After=network-online.target local-fs.target
+
+[Service]
+Type=simple
+WorkingDirectory=/srv/dmqnode/dmqnode-%i/
+ExecStart=/usr/sbin/dmqnode -c /srv/dmqnode/dmqnode-%i/etc/config.ini
+ExecReload=/bin/kill -HUP $MAINPID
+
+Restart=on-failure
+User=dmqnode
+Group=core
+
+# Could be even 1000 (to disable oom killing),
+# but set to -950 for the OOM killer to be able to kill an instance
+# of DMQ node if the server is about to die.
+OOMScoreAdjust=-950
+
+# Note infinity is used instead of unlimited
+LimitNOFILE=100000:100000
+LimitMEMLOCK=infinity:infinity
```

Comment:
Oh, you don't need this! I'll adjust the guide.

Reply:
(it's relevant to dhtnode, not for other apps)

Reply:
Ha yes. Too much copypasta! :D

Reply:
Updated.